### DISCUSSION TITLE
Проблема при запуске командой: npm start

### DISCUSSION THREAD

**Discussion by 0010513** [MrZillaGold/VK2Discord#520] [Q&A] [answered]
сделал все по инструкции наченаю запускать и вылетает ошибка 
![2022-05-29_11-30-32](https://user-images.githubusercontent.com/41306823/170859336-9c24c851-052f-4928-8165-c7be0da0356b.png)

**Comment by egorprnn** [comment, marked as answer]
Используйте версии из [релизов](https://github.com/MrZillaGold/VK2Discord/releases/latest), а не из корня репозитория.
Также можете самостоятельно собрать проект используя `npm run-script build`
